docs/info/long-range.md

```diff
@@ -35,6 +35,7 @@ Anyone can add an entry to the table, and entries should include the:
 ### 25mW
 | Max Dist. | Freq | Pkt Rate | TX Power | Type | Failsafe at Max Range? | Pilot Handle | Link to DVR |
 | ---- | -------- | -------- | --------- | --------- | ---------------------- | ------------ | ----------- |
+| 4.6Km | 2.4G | 50HZ | 25mW | Quad | No | febbyeka | [Link](https://www.youtube.com/watch?v=jtiu6GdsKhA) |
```

Comment:
use Hz for frequency  (instead of HZ)

Reply:
HZ is what every other entry uses at the moment, so not ariefadha's fault